FEDERATION.md

```diff
@@ -0,0 +1,47 @@
+# Federation
+
+*This describes Gitea's future federation capabilities, not what it can do currently.*
+
+Gitea is federated using [ActivityPub](https://www.w3.org/TR/activitypub/) and the [ForgeFed extension](https://forgefed.org/) so you can interact with users and repositories from other instances as if they were on your own instance. By using the standardized ActivityPub protocol, users on any fediverse software such as [Mastodon](https://joinmastodon.org/) can follow Gitea users, stars repos, receive activity updates, and comment on issues.
```

Comment:
Grammatical errors:
In the [simple present](https://en.wikipedia.org/wiki/Simple_present) tense, with [modal verbs](https://en.wikipedia.org/wiki/Modal_verb), use the [infinitive](https://en.wikipedia.org/wiki/Infinitive) form of the main verb without “to.”

```suggestion
Gitea is federated using [ActivityPub](https://www.w3.org/TR/activitypub/) and the [ForgeFed extension](https://forgefed.org/) so you can interact with users and repositories from other instances as if they were on your own instance. By using the standardized ActivityPub protocol, users on any fediverse software such as [Mastodon](https://joinmastodon.org/) can follow Gitea users, star repos, receive activity updates, and comment on issues.
```